pallets/domains/src/types.rs

```diff
@@ -0,0 +1,92 @@
+use frame_support::pallet_prelude::*;
+use frame_support::traits::Currency;
+use sp_runtime::traits::Zero;
+
+use pallet_parachain_utils::WhoAndWhen;
+
+use super::*;
+
+pub(crate) type DomainName<T> = BoundedVec<u8, <T as Config>::MaxDomainLength>;
+pub(crate) type DomainsVec<T> = BoundedVec<DomainName<T>, <T as Config>::MaxDomainsPerAccount>;
+pub(crate) type InnerValue<T> = Option<DomainInnerLink<<T as frame_system::pallet::Config>::AccountId>>;
+pub(crate) type OuterValue<T> = Option<BoundedVec<u8, <T as Config>::OuterValueLimit>>;
```

Comment:
Not consistent naming:
`MaxDomainLength` vs `OuterValueLimit`

This will be consistent: `MaxOuterValueLength`